.github/workflows/pr.yaml

```diff
@@ -148,9 +148,19 @@ jobs:
           LXD_CHANNEL=$(echo '${{ matrix.lxd_channel }}' | sed 's#/#-#g')
           echo "LXD_CHANNEL=$LXD_CHANNEL" >> $GITHUB_OUTPUT
 
+      - name: Create OIDC users
+        if: ${{ matrix.lxd_channel != '5.0/edge' }}
+        shell: bash
+        run: ./tests/scripts/create_oidc_identities
+
       - name: Run Playwright tests
         run: npx playwright test --project ${{ matrix.browser }}:lxd-${{ steps.lxd-env.outputs.LXD_CHANNEL }}
 
+      - name: Delete OIDC users
```

Comment:
Probably we don't need this clean-up in the workflow, we also don't call it in the coverage flow?

Reply:
Yeah this is more needed for local testing, for cleanup purposes. Not needed in workflows as you mentioned :+1: